### ISSUE TITLE
Create save and load functionalities

### ISSUE THREAD

**Issue by giuseppetotaro** [memex-explorer/sce-domain-discovery#7]
The users should be able to save and load a model on the server. For instance, the user can save a model on the server by giving a name and load it subsequently. All the saved models must be reported in a list where the users can select the model to be used.

**Comment by wmburke** [comment]
I'd like to amend the way you have laid this out, Giuseppe. Consider the following:
1. Whenever a new model is created (including the first time the page is loaded), a new model is saved to disk with the name of the first query that is run using it. Every time the model is updated, it's also saved to disk. The name of this model shows up in the left hand column, above the Create a New Model link and has like a pink background or something to make it obvious it's the current model.
2. If they click Create a New Model, it moves the current model to a list of models in the left hand column that is below the other stuff in the column, and a new blank model is formed.
3. Clicking on a model in the list of models makes that one the current model, moving its name to the top of the column and moving whatever the current model is down to the list of models.
4. The list of models is alphabetized.
5. Retain import/export functionality. We can make it more slick later, but it's fine for now. The current model can be exported/imported to.
6. In an ideal world, they should be able to change the name of the model. If this is difficult, it is low priority and a new issue should be created for it once the rest of this one is complete.
